packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/generatedAnswer.css

```diff
@@ -0,0 +1,27 @@
+.generated-answer__card-header {
+  justify-content: space-between;
+  align-items: center;
```

Comment:
I believe you can use `slds-grid_vertical-align-center` here no?

Reply:
I would still need this class for the `justify-content> space-between` so between using two classes on my element where each class provides one CSS rule and using one class that provides both rules I prefer the latter option for simplicity.